opennms-config/src/main/java/org/opennms/netmgt/config/VacuumdConfigFactory.java

```diff
@@ -116,18 +112,10 @@ public static synchronized void init() throws IOException {
             return;
         }
 
-        InputStream is = null;
-
-        try {
-            is = new FileInputStream(ConfigFileConstants.getFile(ConfigFileConstants.VACUUMD_CONFIG_FILE_NAME));
-            setInstance(new VacuumdConfigFactory(is));
-        } finally {
-            if (is != null) {
-                IOUtils.closeQuietly(is);
-            }
-        }
-        
-        m_loadedFromFile = true;
+        ConfigurationService configurationService = new ConfigurationService(); // TODO: Patrick dependency injection?
```

Comment:
@j-white I am not sure how we can get here (place and time) access to a spring bean. Lets discuss

Reply:
If we continue to use the singleton pattern, we'll run into problems with DI and access to other services (beans).